ApplicationLibCode/ProjectDataModel/RimStatisticsContourMap.cpp

```diff
@@ -535,6 +597,36 @@ std::vector<int> RimStatisticsContourMap::selectedTimeSteps() const
     return steps;
 }
 
+//--------------------------------------------------------------------------------------------------
+///
+//--------------------------------------------------------------------------------------------------
+std::vector<QString> RimStatisticsContourMap::selectedFormations() const
+{
+    return m_selectedFormations();
+}
+
+//--------------------------------------------------------------------------------------------------
+///
+//--------------------------------------------------------------------------------------------------
+std::vector<std::vector<cvf::Vec3d>> RimStatisticsContourMap::selectedPolygons()
+{
+    std::vector<std::vector<cvf::Vec3d>> allLines;
```

Comment:
This is a lot of text `std::vector<std::vector<cvf::Vec3d>` 
Consider adding a `using` statement in `RigPolyLinesData`.

Suggested naming
`using PolyLineCoords = std::vector<std::vector<cvf::Vec3d>>;`